packages/plugins/rubygem-unf_ext/rubygem-unf_ext.spec

```diff
@@ -0,0 +1,98 @@
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name unf_ext
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.7.2
+Release: 1%{?dist}
+Summary: Unicode Normalization Form support library for CRuby
+Group: Development/Languages
+License: MIT
+URL: https://github.com/knu/ruby-unf_ext
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby-devel
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+# end specfile generated dependencies
+
+%description
+Unicode Normalization Form support library for CRuby.
+
+
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{pkg_name}.
+
+%prep
+%{?scl:scl enable %{scl} - << \EOF}
+gem unpack %{SOURCE0}
+%{?scl:EOF}
+
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+%{?scl:scl enable %{scl} - << \EOF}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:EOF}
+
+%build
+# Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
+gem build %{gem_name}.gemspec
+%{?scl:EOF}
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install
+%{?scl:EOF}
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+mkdir -p %{buildroot}%{gem_extdir_mri}
+cp -a .%{gem_extdir_mri}/{gem.build_complete,*.so} %{buildroot}%{gem_extdir_mri}/
+
+# Prevent dangling symlink in -debuginfo (rhbz#878863).
+rm -rf %{buildroot}%{gem_instdir}/ext/
+
+%files
+%dir %{gem_instdir}
+%exclude %{gem_instdir}/ext
```

Comment:
@apuntamb,
Koji build failed with below error related to `unf_ext` - 

```

Processing files: tfm-rubygem-unf_ext-0.0.7.2-1.el7.x86_64
BUILDSTDERR: warning: File not found: /builddir/build/BUILDROOT/tfm-rubygem-unf_ext-0.0.7.2-1.el7.x86_64/opt/theforeman/tfm/root/usr/share/gems/gems/unf_ext-0.0.7.2/ext
BUILDSTDERR: error: File not found: /builddir/build/BUILDROOT/tfm-rubygem-unf_ext-0.0.7.2-1.el7.x86_64/opt/theforeman/tfm/root/usr/share/gems/gems/unf_ext-0.0.7.2/ext
RPM build errors:
BUILDSTDERR:     File not found: /builddir/build/BUILDROOT/tfm-rubygem-unf_ext-0.0.7.2-1.el7.x86_64/opt/theforeman/tfm/root/usr/share/gems/gems/unf_ext-0.0.7.2/ext
BUILDSTDERR:     File not found: /builddir/build/BUILDROOT/tfm-rubygem-unf_ext-0.0.7.2-1.el7.x86_64/opt/theforeman/tfm/root/usr/share/gems/gems/unf_ext-0.0.7.2/ext
Child return code was: 1
EXCEPTION: [Error()]
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/mockbuild/trace_decorator.py", line 96, in trace
    result = func(*args, **kw)
  File "/usr/lib/python2.7/site-packages/mockbuild/util.py", line 634, in do
    raise exception.Error("Command failed: \n # %s\n%s" % (command, output), child.returncode)
Error: Command failed: 
 # bash --login -c /usr/bin/rpmbuild -bb --target x86_64 --nodeps /builddir/build/SPECS/rubygem-unf_ext.spec
```
I think, you will need this `%{gem_instdir}/ext`.

Reply:
As per line 71 i.e `rm -rf %{buildroot}%{gem_instdir}/ext/`, you can actually delete these lines 75 & 80.

@ekohl, what do you think?

Reply:
@kgaikwad Deleting lines- 75 and 80, works.
@ekohl Could you please take a look?